helm/prometheus-meta-operator/values.yaml

```diff
@@ -14,6 +14,7 @@ pod:
     id: 1000
 
 alertmanager:
+  enabled: false
```

Comment:
Why is that not true?

Reply:
We need 2 releases: 
- a release to delete alertmanager ownership from PMO
- another release that inlcudes alertmanager in Helm